ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -226,12 +226,12 @@ open class PeoplePicker: BadgeField {
         setNeedsLayout()
     }
 
-    private func hidePersonaSuggestions() {
+    open func hidePersonaSuggestions() {
         personaSuggestionsView.removeFromSuperview()
         containingViewBoundsObservation = nil
     }
 
-    private func getSuggestedPersonas() {
+    open func getSuggestedPersonas() {
```

Comment:
>getSuggestedPersonas [](start = 14, length = 20)

it's not very clear to me why we need to make public this and the method above, they seem like very specific implementation details. What should a client expect if they call these on an instance?

Reply:
You are right. This one not needed. reverting this change